packages/rum-core/src/domain/rumEventsCollection/view/trackViewFocus.ts

```diff
@@ -0,0 +1,91 @@
+import {
+  elapsed,
+  ClocksState,
+  preferredNow,
+  addEventListener,
+  DOM_EVENT,
+  Duration,
+  clocksNow,
+  preferredClock,
+  PreferredTime,
+} from '@datadog/browser-core'
+
+const MAX_NUMBER_OF_FOCUSED_TIME = 500
```

Comment:
Could you add a comment to explain why 500?
"arbitrary value" is an acceptable explanation